### PR TITLE
Feature/deployment

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

File: src/server/index.js [hunk 1]
```diff
@@ -143,16 +139,6 @@ if (env === 'development') {
   ]);
 }
 
-if (env === 'production') {
```
Comment (r4vi) [inline src/server/index.js]: @adamgraves-choices I'd suggest doing a new commit that just removes this section (L146-L155)

File: src/server/index.js [hunk 2]
```diff
@@ -16,16 +16,12 @@ const manifest = {
...
-    },
-    load: {
-      sampleInterval: 5
     }
   },
   connections: [
     {
       port: port,
       routes: {
-        security: !debug,
```
Comment (r4vi) [inline src/server/index.js]: this needs to stay

File: src/server/index.js (hunk 1 above)
Comment (adamgraves-choices) [inline src/server/index.js]: Will do @r4vi. I'll rebase at some point.

File: src/server/index.js (hunk 2 above)
Comment (adamgraves-choices) [inline src/server/index.js]: Thanks @r4vi. Added it back. What does this do just out of interest?

**Comment by r4vi** [comment]
That's the part that turns on security http headers like x-frame-options etc

**Comment by adamgraves-choices** [comment]
@leonardpowellNHSD I've made the changes that Ravi has requested. if you want to merge it into master, we can get a deployment done.

**Pull request merged**